trunk/SUAVE/Methods/Aerodynamics/AVL/Data/Settings.py

```diff
@@ -16,60 +17,61 @@
 
 class Settings(Data):
 
-	def __defaults__(self):
-		
-		self.run_cases = Run_Case.Container()
-		self.filenames = Data()
-		self.flow_symmetry = Data()
-		self.discretization = Data()
-		
-		self.num_control_surfaces = 0
-		
-		self.discretization.defaults = Data()
-		self.discretization.surfaces = Data()
-		self.discretization.defaults.wing = AVL_Discretization_Settings()
-		self.discretization.defaults.fuselage = AVL_Discretization_Settings()
-		self.discretization.defaults.fuselage.spanwise_spacing = 3
-		self.discretization.defaults.fuselage.spanwise_spacing_scheme = 'equal'
-		self.discretization.defaults.fuselage.nose_interpolation = 'parabolic'
-		self.discretization.defaults.fuselage.tail_interpolation = 'linear'
-		
-		self.filenames.avl_bin_name    = 'avl' # to call avl from command line. If avl is not on the system path, include absolute path to the avl binary
-		self.filenames.run_folder      = 'avl_files' # local reference, will be attached to working directory from which avl was created
-		self.filenames.features        = 'aircraft.avl'
-		self.filenames.batch_template  = 'batch_{0:03d}.run'
-		self.filenames.deck_template   = 'commands_{0:03d}.deck'
-		self.filenames.output_template = 'results_{}.txt'
-		self.filenames.case_template   = 'case_{0:03d}_{1:02d}'
-		self.filenames.log_filename    = 'avl_log.txt'
-		self.filenames.err_filename    = 'avl_err.txt'
-		
-		#------------------------------------------
-		# 1:  Symmetry about the plane
-		# -1: Antisymmetry (Cp constant on plane)
-		# 0:  Symmetry not guaranteed
-		#------------------------------------------
-		self.flow_symmetry.xz_plane = 0	# Symmetry across the xz-plane, y=0
-		self.flow_symmetry.xy_parallel = 0 # Symmetry across the z=z_symmetry_plane plane
-		self.flow_symmetry.z_symmetry_plane = 0.0
-		
+        def __defaults__(self):
+
+                self.run_cases = Run_Case.Container()
+                self.filenames = Data()
+                self.flow_symmetry = Data()
+                self.discretization = Data()
+
+                self.num_control_surfaces = 0
+
+                self.discretization.defaults = Data()
+                self.discretization.surfaces = Data()
+                self.discretization.defaults.wing = AVL_Discretization_Settings()
+                self.discretization.defaults.fuselage = AVL_Discretization_Settings()
+                self.discretization.defaults.fuselage.spanwise_spacing = 3
+                self.discretization.defaults.fuselage.spanwise_spacing_scheme = 'equal'
+                self.discretization.defaults.fuselage.nose_interpolation = 'parabolic'
+                self.discretization.defaults.fuselage.tail_interpolation = 'linear'
+
+                self.filenames.avl_bin_name    = 'C:/Users/Matthew/Documents/AVL/avl.exe' # previously 'avl' # to call avl from command line. If avl is not on the system path, include absolute path to the avl binary
```

Comment:
I think leaving as 'avl' might be best, since ideally the user will have avl in their path. Then, if it doesn't work there should be a meaningful error message.